pkg/interop/native/roles/roles.go

```diff
@@ -20,6 +20,7 @@ type Role byte
 const (
 	StateValidator Role = 4
 	Oracle         Role = 8
+	NeoFSAlphabet  Role = 16
```

Comment:
We need to add it to `TestRoleManagementRole` also.

Reply:
Fixed!